packages/cli/src/constructs/check-group.ts

```diff
@@ -133,7 +133,15 @@ export class CheckGroup extends Construct {
     this.privateLocations = props.privateLocations
     this.concurrency = props.concurrency
     this.apiCheckDefaults = { ...defaultApiCheckDefaults, ...props.apiCheckDefaults }
+
     this.environmentVariables = props.environmentVariables ?? []
+    this.environmentVariables.forEach(ev => {
+      // only empty string is checked because the KeyValuePair.value doesn't allow undefined or null.
+      if (ev.value === '') {
```

Comment:
`value` could also be null or undefined. Do you know if our Joi validation throws an error in that case?

Reply:
I looked, and it seems like `null` is allowed. I wonder if we should just allow empty strings in the API.

Reply:
@clample , `null` is not allowed to be used a `KeyValuePair.value`, just `string`. So, the only 'falsy' value is the empty string `''`.